src/main/scala/fundamentals/level03/TryExercises.scala

```diff
@@ -58,6 +58,19 @@ object TryExercises {
     **/
   def parseBooleanSafe(str: String): Try[Boolean] = ???
 
+
+  /**
+    * scala> increment("10")
+    * = Right(11)
+    *
+    * scala> increment("NaN")
+    * = Left(TryError("NaN is not a valid number"))
```

Comment:
Should these be `Success` and `Failure` instead of `Left` and `Right`? Or where you going to convert it to an `Either` ? The return type is `Try[Int]`

Reply:
I actually copied the Right/Left from previous example which is wrong I think. Will fix it. Converting to Either is next problem so for this I will leave it as `Try[Int]`